hosts

```diff
@@ -3,6 +3,7 @@ bloggulus.com
 
 [coffee]
 daileybrue.coffee
+whosbuying.coffee
```

Comment:
So this won't actually need to be here because this change (of adding another domain to point at an existing server) doesn't require a new host.